### PR TITLE
Fork/support arm64

### PR DESCRIPTION
**Pull request by Darin726**
Make weex Support arm64-v8a

### CODE REVIEW

File: android/sdk/build.gradle [hunk 1]
```diff
@@ -114,16 +127,32 @@ android {
 //        include 'weex-rax-extra-api.js'
 //    }
 
+    if(project.hasProperty('asfRelease')){
```
Comment (YorkShen) [inline android/sdk/build.gradle]: The `if(project.hasProperty('asfRelease'))` block is not needed anymore, you can delete it.

File: android/sdk/build.gradle [hunk 2]
```diff
@@ -79,9 +87,14 @@ if (!project.hasProperty('ignoreVersionCheck') || !project.getProperty('ignoreVe
 
 android {
     compileSdkVersion project.compileSdkVersion
-    buildToolsVersion project.buildToolsVersion
+//    buildToolsVersion project.buildToolsVersion
     resourcePrefix "weex"
     useLibrary 'org.apache.http.legacy'
+    packagingOptions {
```
Comment (YorkShen) [inline android/sdk/build.gradle]: In order to make Android App with or without `libc++_shared.so` run as expected, you need to do the following things:
1. delete `libc++_shared.so` in `libs` directory
1. Add a command flag `./gradlew -PIgnoreLibC=true` to enable `packagingOptions`. The `packagingOptions` should be disabled as default.

File: android/playground/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -128,15 +128,15 @@ under the License.
...
-        <provider
-            android:name="android.support.v4.content.FileProvider"
-            android:authorities="com.alibaba.weex.fileprovider"
-            android:grantUriPermissions="true"
-            android:exported="false">
-            <meta-data
-                android:name="android.support.FILE_PROVIDER_PATHS"
-                android:resource="@xml/filepaths" />
-        </provider>
+        <!--<provider-->
```
[the remark this answers is not in the document]
Comment (Darin726) [reply android/playground/app/src/main/AndroidManifest.xml]: this will make playground crash

**Comment by wuhh2016** [comment]
什么时候发布？

**Pull request merged**